### ISSUE TITLE
Accessibility / content audit

### ISSUE THREAD

**Issue by zachberry** [ucfopen/Obojobo#1003]
Idea came up during a presentation, but it would be nice to aggregate all of the accessibility warnings (and possibly missing content like Figures with no image) and allow authors to pull that audit at any time
